core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.html

```diff
@@ -79,7 +79,7 @@ <h3>Reload a single collection</h3>
   <div class="container" style="margin-left: 0;">
     <h3>Load dummy new structures data (only admins)</h3>
     <div class="row">
-      <span class="col-lg-4 col-md-4 col-sm-4">
+      <span class="col-lg-8 col-md-8 col-sm-8">
```

Comment:
Can you share the UI of the whole Activity tab after these changes?

Reply:
Hey @DubeySandeep This only affects the load new strutures tab, of which the sceenshot i have already applied.

Reply:
This is how it looks on develop currently.
![image](https://user-images.githubusercontent.com/30312043/87310859-29e7fb00-c53c-11ea-9491-2e1c79316a8a.png)